### ISSUE TITLE
Is this active, maintained?

### ISSUE THREAD

**Issue by mm108** [evothings/evothings-studio#190]
just curious, is Evothings being maintained and updated?

**Comment by ppetree** [comment]
Doesn't look like it.  No updates in 2 years and no response to your query and you asked this nearly 2 months ago.

**Comment by mm108** [comment]
Okay thanks :)

**Comment by gokr** [comment]
Currently, no, it's not actively maintained but the servers are still operated. Exactly what will happen is not decided.

**Comment by ppetree** [comment]
@gokr There's certainly a need within the phonegap/cordova market.  To easily edit the config.xml (switches for all the various flags - on is in config, off is removed), add plugins, etc.  To run the code in the emulator.  To easily maintain keystores and certificates. To package and upload to phonegap build.  There's a ton of features that would make the product very viable.

There's a lot of moving pieces that can be installed, configured, updated and generally managed for users.  To run node, npm and cli from the command line is archaic. (I did that crap in the 80's!).

Visual Studio sucks beyond sucking... they're so far behind on the CLI that it may as well be considered abandoned - not to mention all the bugs they have.  

The others are all trying to serve too many masters and as a result, cordova gets left behind. Intel was there with their XDK but they abandoned that two years ago because of the build servers.  They let the product run on for awhile allowing you to package for PGB but they eventually killed the entire product as they were focusing on IoT because that sells chips.

**Comment by gokr** [comment]
Yeah, so... the jury is still out a bit on what we should do with Evothings Studio. I haven't been attentive on gitter lately, but I will be there if anyone wants to discuss this.

**Comment by sanyaade-teachings** [comment]
@gokr 

It will be nice to document how to use a local server in replace for https://deploy.evothings.com 

There is still a lot of interests in Evothings Studio most especially those who have been following from MoSync and Reload era.

Many thanks to you all for all the efforts that had gone into Evothings.

Hear from you soon!

God blesses!!!

Best regards,
Sanyaade

**Comment by mazekeeper** [comment]
It all comes from the HyperReload project, by Mikael Kindborg, where the idea was to run locally in the first place. Problem was the client isolation (aka AP isolation) in corporate LANs where you couldn't get to one computer to another, why we made the proxy. That can surely be used as a local back-end, it also hasn't been touched for a long time but then again it it stable and has very few moving parts.  https://github.com/mikaelkindborg/HyperReload

**Comment by sanyaade-teachings** [comment]
@mazekeeper 

Many thanks for your quick reply.

Mikael Kindborg HyperReload was originally Ruby based and then Nodejs/Cordova.

The Client isolation can be eliminated if we adopt socket layer. I have looked into PhoneGap Desktop (https://github.com/phonegap/phonegap-app-desktop), Ionic Studio(https://ionicframework.com/studio) to see how things are done in those platforms and ideas can help in getting Evothings beyond. Network will always be a bottleneck but can be reduced to peer-to-peer (websocket factor)

Are you still working on Evothings platforms?

We can repackage Evothings local CI/CD or combine the Studio with Reload as All-in-One suite?

From Mikael posts and on his websites, it seems is moving into another avenue now. Good man!

Gokr said that jury are out, so Evothings may not be developed further. It could still be used in teachings IoT, Robotics, Research and many more...

I posted on Gitter but no replies. I will keep examine and hacking Evothings platforms.

Hear from you soon!

God blesses!!!

Best regards,
Sanyaade

**Comment by sanyaade-teachings** [comment]
@mazekeeper 

Many thanks for your quick reply.

Mikael Kindborg HyperReload was originally Ruby based and then Nodejs/Cordova.

The Client isolation can be eliminated if we adopt socket layer. I have looked into PhoneGap Desktop (https://github.com/phonegap/phonegap-app-desktop), Ionic Studio(https://ionicframework.com/studio) to see how things are done in those platforms and ideas can help in getting Evothings beyond. Network will always be a bottleneck but can be reduced to peer-to-peer (websocket factor)

Are you still working on Evothings platforms?

We can repackage Evothings local CI/CD or combine the Studio with Reload as All-in-One suite?

From Mikael posts and on his websites, it seems is moving into another avenue now. Good man!

Gokr said that jury are out, so Evothings may not be developed further. It could still be used in teachings IoT, Robotics, Research and many more...

I posted on Gitter but no replies. I will keep examine and hacking Evothings platforms.

Hear from you soon!

God blesses!!!

Best regards,
Sanyaade
Duplicate of #
